cluster/kube/client_ingress.go

```diff
@@ -68,6 +68,12 @@ func kubeNginxIngressAnnotations(directive ctypes.ConnectHostnameToDeploymentDir
 		}
 	}
 
+	if env["AKASH_PROVIDER_ISSUER_TYPE"] == "cluster-issuer" {
+		result["cert-manager.io/cluster-issuer"] = env["AKASH_PROVIDER_ISSUER_NAME"]
+	} else if env["AKASH_PROVIDER_ISSUER_TYPE"] == "issuer" {
+		result["cert-manager.io/issuer"] = env["AKASH_PROVIDER_ISSUER_NAME"]
+	}
+
```

Comment:
would prefer to have these as constants, similar to [this](https://github.com/akash-network/provider/blob/e7aa0b5b81957a130f1dc584f335c6f9e41db6b1/cluster/kube/builder/netpol.go#L39-L40)